vignettes/convert.qmd

```diff
@@ -0,0 +1,63 @@
+---
+title: "Convert an existing project"
```

Comment:
just a thought: once this is merged we might ask the #multilingual channel whether anyone wants to give the docs a read

Reply:
👍